### PR TITLE
Waydroid: new, 1.4.3

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

- waydroid: new, 1.4.3
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>
- pyclip: new, 0.7.0
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>
- gbinder-python: new, 1.1.2
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>
- libgbinder: new, 1.1.40
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>
- libglibutil: new, 1.0.79
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>

Package(s) Affected
-------------------

- gbinder-python: 1.1.2
- libgbinder: 1.1.40
- libglibutil: 1.0.79
- pyclip: 0.7.0
- waydroid: 1.4.3

Security Update?
----------------

No

Build Order
-----------

```
#buildit libglibutil libgbinder gbinder-python pyclip waydroid
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by OriginCode** [comment]
LGTM on amd64:
圖片

File: app-emulation/waydroid/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,10 @@
+PKGNAME=waydroid
+PKGSEC=utils
+PKGDEP="systemd nftables pygobject-3 gtk-3 dbus-python dnsmasq lxc iproute2 \
+        python-3 gbinder-python pyclip"
+PKGDES="An Android container manager"
+
+ABHOST=noarch
+
+# It's just python
```
Comment (MingcongBai) [inline app-emulation/waydroid/autobuild/defines]: Then this should not be needed. Autobuild will not attempt to split out -dbg if the package is marked as noarch.

File: app-emulation/waydroid/autobuild/patches/0001-fedora-no-apparmor.patch [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+@@ -161,9 +161,6 @@ def set_lxc_config(args):
+     tools.helpers.run.user(args, command)
+     command = ["cp", "-fpr", seccomp_profile, lxc_path + "/waydroid.seccomp"]
+     tools.helpers.run.user(args, command)
+-    if get_apparmor_status(args):
+-        command = ["sed", "-i", "-E", "/lxc.aa_profile|lxc.apparmor.profile/ s/unconfined/{}/g".format(LXC_APPARMOR_PROFILE), lxc_path + "/config"]
+-        tools.helpers.run.user(args, command)
+ 
+     nodes = generate_nodes_lxc_config(args)
+     config_nodes_tmp_path = args.work + "/config_nodes"
```
Comment (MingcongBai) [inline app-emulation/waydroid/autobuild/patches/0001-fedora-no-apparmor.patch:37]: Would recommend putting this in AOSC-Tracking (optional but recommended).

File: lang-python/gbinder-python/autobuild/prepare [hunk 3]
```diff
@@ -0,0 +1,2 @@
+abinfo "Enabling cython in setup.py ..."
+sed -i "/^USE_CYTHON =/s/False/True/" "$SRCDIR"/setup.py
```
Comment (MingcongBai) [inline lang-python/gbinder-python/autobuild/prepare]: Please use a patch for this purpose, sed is not predictable.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by Cyanoxygen** [comment]
LGTM on amd64:

image

And, Please stop any DNS related services before starting WayDroid!

**Review by MingcongBai — APPROVED** [review]
Tested good on AMD64:

图片

**Comment by hanyuwei70** [comment]
can't start on d2000 (aarch64)
full log: https://fars.ee/iccu

**Pull request merged**